frontend/src/app/static/about-us.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="container my-5">
+  <h1 class="my-3">O nas</h1>
+  <div>
+    <div class="paragraph">Lubisz nowe wyzwania? Chcesz pomagać innym? Zależy Ci na rozwoju? Szukasz wolontariuszy?
+      Jeśli
```

Comment:
What's a reason of this wired formatting? Can You make it more like similar line length paragraphs?